cmd/bbolt/main.go

```diff
@@ -1050,9 +1051,23 @@ func (cmd *benchCommand) Run(args ...string) error {
 	}
 
 	// Print results.
-	fmt.Fprintf(cmd.Stderr, "# Write\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", writeResults.CompletedOps(), writeResults.Duration(), writeResults.OpDuration(), writeResults.OpsPerSecond())
-	fmt.Fprintf(cmd.Stderr, "# Read\t%v(ops)\t%v\t(%v/op)\t(%v op/sec)\n", readResults.CompletedOps(), readResults.Duration(), readResults.OpDuration(), readResults.OpsPerSecond())
+	if options.GoBenchOutput {
+		// below replicates the output of testing.B benchmarks, e.g. for external tooling
+		maxLen := 5 // the length of "Write"
+		gobenchResult := testing.BenchmarkResult{}
+		gobenchResult.T = writeResults.Duration()
+		gobenchResult.N = int(writeResults.CompletedOps())
+		fmt.Fprintf(cmd.Stderr, "%-*s\t%s\n", maxLen, "Write", gobenchResult.String())
```

Comment:
Not sure whether `benchstat` requires the name must be prefixed with "Benchmark"
```suggestion
		fmt.Fprintf(cmd.Stderr, "%-*s\t%s\n", maxLen, "BenchmarkWrite", gobenchResult.String())
```

Reply:
added